scripts/execute-deepnote.py

```
@@ -0,0 +1,891 @@
+        filters={repr(json.dumps(chart_block['filters']))}
+    )
+    # Display the chart
+    display(_chart)
+    print(f"✓ Chart created from {{len({chart_block['var_name']})}} rows")
+except Exception as e:
+    print(f"✗ Chart execution failed: {{type(e).__name__}}: {{e}}")
+    raise
+"""
+                
+                # Execute the chart code
+                msg_id = client.execute(chart_code, silent=False)
```

Ruff (0.14.3)

Local variable `msg_id` is assigned to but never used

Remove assignment to unused variable `msg_id`

(F841)